docs/language/operators.md

```diff
@@ -990,35 +990,39 @@ zq -z 'County=="Los Angeles" | tail 5' schools.zson
 
 ---
 
-## `traverse`
+## `over`
 
 ```
-traverse foo <collection> [=> (seq <sequence>)]
+over <expr> [, <expr>...]
 ```
 
-`traverse` emits each element in collection (i.e., values of type `array`,
-`set`, `record`\*, `map`\*) `foo` as a separate value to child
-nodes in the flowgraph. If the optional sequence `seq`\*\* argument is provided the
-elements of `foo` are read into `seq` as a separate stream sending an `EOS` when
-the last element is traversed. The each element can be accessed inside `seq` by
-referencing the `this` root record.
+For each input value,
+`over` yields a sequence of output values terminated by `EOS`
```

Comment:
EOS is an implementation detail that does not appear anywhere else in the documentation. I don't think we should confuse the reader with it.

Reply:
I agreed and copied what nibs had.  We need to document batching.  Let's leave this here for now since we are redoing a lot in the next step.